stylesheets/styles.css

```diff
@@ -208,6 +229,33 @@ a.user {
   "footer footer";
 }
 
+
+.cart-wrapper {
+  display: grid;
+  width: 100%;
+  grid-gap: 1.5rem;
+  grid-template-columns: 13rem 1fr;
+  grid-template-areas: 
+  "header header"
+  "main main"
+  "main main"
+  "main main"
+  "footer footer";
+}
+
+.wishlist-wrapper {
+  display: grid;
+  width: 100%;
+  grid-gap: 1.5rem;
+  grid-template-columns: 13rem 13rem 13rem 13rem 1fr;
```

Comment:
you can use repeat here just a suggestion

Reply:
Thanks. I will replaced with repeat.